.github/workflows/fair-software.yml

```diff
@@ -0,0 +1,15 @@
+name: fair-software
```

Comment:
This action will execute [this pipeline](https://github.com/fair-software/howfairis-github-action/blob/main/action.yml) on every push - however this is not automatically update the badge.

> This GitHub action will suggest a badge visualizing compliance with the FAIR Software recommendations as described on [fair-software.eu](https://fair-software.eu/).

I don't think this action is not needed, specially not on every push, this badge is something we can manually update.

Reply:
What would you then suggest then?
- Remove this action ?
- Update manually ?
or
```
on:
  pull_request:
    branches:
      - main
``` 
?

Reply:
I'd just remove it